src/qibolab/instruments/qm/program/acquisition.py

```diff
@@ -14,19 +14,22 @@
     AveragingMode,
     ExecutionParameters,
 )
-from qibolab.result import collect
 
 
-def _split(data, npulses, iq=False):
+def _collect(i, q, npulses):
+    """Collect I and Q components of signal."""
+    signal = np.stack([i, q])
+    return np.moveaxis(signal, 0, -1 - int(npulses > 1))
```

Comment:
Maybe add a tiny comment, the usage of boolean as integers are a bit cryptic.

Reply:
Added a comment in 11028db21587ce218ce72fb14d638bcfce66cd0c.